build_daemon/example/example.dart

```diff
@@ -9,10 +9,17 @@ import 'package:build_daemon/client.dart';
 
 void main(List<String> args) async {
   BuildDaemonClient client;
-  var workingDirectory = Directory.current.path;
+  var workingDirectory =
+      Directory.fromUri(Uri(path: Directory.current.path + '/../example')).path;
+
   try {
-    client = await BuildDaemonClient.connect(workingDirectory,
-        ['pub', 'run', 'build_daemon', workingDirectory, 'some-option']);
+    client = await BuildDaemonClient.connect(workingDirectory, [
+      'pub',
+      'run',
+      'build_runner',
+      'daemon',
+      '--delete-conflicting-outputs',
```

Comment:
This isn't defaulted today so this would be a change from current behavior

Reply:
This is just an example.